predicate/predicate.py

```diff
@@ -1,53 +1,17 @@
 import re
 
+from django.db.models.manager import Manager
 from django.db.models.query_utils import Q
+from django.utils.datastructures import SortedDict
```

Comment:
`SortedDict` has been [deprecated as of Django 1.7, and is now removed in Django 1.9.](https://code.djangoproject.com/wiki/SortedDict)  Django<1.7 is no longer supported by django-core, and Django 1.8+ only runs on Python 2.7+, where `collections.OrderedDict` shoudl be a drop-in replacement for `SortedDict`.